info.json

```diff
@@ -0,0 +1,223 @@
+                "name": "ZTP Profiles",
+                "apiName": "z_t_p_profiles",
+                "views": [
+                    "list",
+                    "detail",
+                    "form"
+                ]
+            }
+        },
+        "views": {
+            "navigation": [
+                "FortiManager"
+            ]
+        },
+        "roles": [
+            {
+                "name": "FortiManager-Playbook"
+            },
+            {
+                "name": "FortiManager-Admin"
+            }
+        ],
+        "playbooks": [
+            {
+                "name": "FortiManager(FMG)-Buttons"
+            },
+            {
+                "name": "FortiManager(FMG)-CustomPlaybooks"
+            },
+            {
+                "name": "FortiManager(FMG)-Microservices"
+            },
+            {
+                "name": "FortiManager(FMG)-PlayGround"
+            },
+            {
+                "name": "FortiManager(FMG)-RecordHandlers"
+            },
+            {
+                "name": "FortiManager(FMG)-Synchronizing"
+            },
+            {
+                "name": "FortiManager(FMG)-Triggers"
+            }
+        ],
+        "globalVariables": [
+            {
+                "id": 7,
+                "name": "Server_fqhn",
+                "value": "10.99.230.22",
+                "default_value": "master.fortisoar.in"
+            }
+        ],
+        "playbookBlocks": [
+            {
+                "name": "Dynamic Manager and Device List"
+            }
+        ],
+        "dashboards": [
+            {
+                "name": "FortiManager Device Assignments - By ZTP Profile"
+            },
+            {
+                "name": "FortiManager Device Config and Policy Manager"
+            },
+            {
+                "name": "FortiManager Device Metadata Assignments and Data"
+            },
+            {
+                "name": "FortiManager Device Metadata Manager"
+            },
+            {
+                "name": "FortiManager Device Metafield Templates"
+            },
+            {
+                "name": "FortiManager Metadata Assignments"
+            },
+            {
+                "name": "FortiManager Product Details"
+            },
+            {
+                "name": "FortiManager ZTPF Docs - Main"
+            },
+            {
+                "name": "FortiManager ZTPF Docs - Metafield Sources"
+            },
+            {
+                "name": "FortiManager ZTPF Docs - ZTP Profiles"
+            },
+            {
+                "name": "FortiManager ZTP Profile and Phase for Devices"
+            },
+            {
+                "name": "FortiManager ZTP Profile and Phase Summary"
+            }
+        ],
+        "connectors": [
+            {
+                "name": "Code Snippet",
+                "apiName": "code-snippet"
+            },
+            {
+                "name": "Fortinet FortiManager Custom",
```

Comment:
@fortijames please mention released released connector here

Reply:
> @fortijames please mention released released connector here

I assume you mean the `Fortinet FortiManager Custom`. This connector is being replaced.